### ISSUE TITLE
 404 Not Found

### ISSUE THREAD

**Issue by topiptv** [amidevous/odiniptvpanelfreesourcecode#1]
sudo wget -O /root/install.py3  https://github.com/amidevous/odiniptvpanelfreesourcecode/raw/master/install.py3 && sudo python3.10 /root/install.py3
--2023-11-13 12:45:36--  https://github.com/amidevous/odiniptvpanelfreesourcecode/raw/master/install.py3
Resolving github.com (github.com)... 140.82.121.3
Connecting to github.com (github.com)|140.82.121.3|:443... connected.
HTTP request sent, awaiting response... 404 Not Found
2023-11-13 12:45:36 ERROR 404: Not Found.

**Comment by amidevous** [comment]
This is normal as I say the installers are being developed I move them to the Install folder

So false change change in the url of each file

master in master/install

In addition, it is to mark in the Readm that it is in developing

and that you install at your risk and danger

So thank you for not opening up any

it will be settled in any day

I already have the binaries

For Ubuntu 18.04/20.04/22.04

Debian 10/11

Centos 7 and 8

The I block on Fedora the minimum OpenSSL patch for php 7.2

Do not work on this distribution I am busy making a new

From the 7.4 Patch from Remi

https://git.remirepo.net/cgit/rpms/php/php74.git/plain/php-7.4.26-openssl3.patch

When it will be done and function I could finish all the debug of the install



In addition for the moment the installer installs the official version encrypt although the source code is available publicly here on Github in the admin folder

Basically I remind you that it is a owner panel which and encrypter

and available only in installation for Ubuntu 18.04

In less than 3 days I decode all the code

and add new distribution in support

so it's normal for everyone to be finished in if can time

I still have between 2 and 5 days of work for everyone to correct

So thank you for your patience
